webclient/app/src/localization/translations-de-DE.js

```diff
@@ -43,7 +43,7 @@ const translationsDeDE = {
     "entityDialog.noRelations": "Keine Relationen angelegt.",
     "entityDialog.noFields": "Keine Felder angelegt.",
     "entityDialog.domainIsLoading": "Domain wird geladen.",
-    "entityDialog.editDomain": "Domain bearbeiten.",
+    "entityDialog.editDomain": "{{entityName}} bearbeiten.",
```

Comment:
its better to call it entity than domain
```suggestion
    "entityDialog.editEntity": "{{entityName}} bearbeiten.",
```